Makefile

```diff
@@ -0,0 +1,27 @@
+# Installation directories.
+PREFIX ?= $(DESTDIR)/usr
+BINDIR ?= $(PREFIX)/bin
+MANDIR ?= $(PREFIX)/share/man
+UNITDIR ?= $(PREFIX)/lib/systemd/system
+DSSLIBDIR ?= $(PREFIX)/lib/docker-storage-setup
+SYSCONFDIR ?= $(DESTDIR)/etc/sysconfig
+PROFILEDIR ?= $(DESTDIR)/etc/profile.d
+
+.PHONY: test
+test:
+	./tests/run-tests.sh
+
+.PHONY: clean
+clean:
+	-rm -rf *~ \#* .#*
+
+.PHONY: install
+install:
+
+	install -D -m 755 docker-storage-setup.sh ${BINDIR}/docker-storage-setup
+	install -D -m 644 docker-storage-setup.service ${UNITDIR}/docker-storage-setup.service
+	install -D -m 644 docker-storage-setup.conf ${DSSLIBDIR}/docker-storage-setup
+	install -D -m 644 docker-storage-setup-override.conf ${SYSCONFDIR}/docker-storage-setup
```

Comment:
Don't like the idea of overwriting user's config file `/etc/sysconfig/docker-storage-setup`. Can't we do this only if /etc/sysconfig/docker-storage-setup is not present.